version

```diff
@@ -1 +1 @@
-4.0.2
\ No newline at end of file
+4.0.5
```

Comment:
forgot the newline... somebody did somewhere earlier as well. Please add it back